src/goto-programs/remove_exceptions.cpp

```diff
@@ -95,6 +96,7 @@ class remove_exceptionst
 protected:
   symbol_tablet &symbol_table;
   std::map<irep_idt, std::set<irep_idt>> &exceptions_map;
+  bool remove_instanceof;
```

Comment:
How about changing this to `run_remove_instanceof` to avoid ambiguity?

Reply:
Called it `remove_added_instanceof` to make clear that it only removes the instanceof calls that it adds itself rather than running `remove_instanceof` over the whole function.